proto/encryption.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package encryption;
+
+import "proto/context.proto";
+
+message SetEncryptionConfigRequest {
+  context.RequestContext request_context = 1;
+
+  bool enabled = 2;
+
+  string key_uri = 3;
+}
```

Comment:
will the client have to set all fields in the request, and we will directly upsert these to the DB? Might be worth mentioning as a comment that all fields are required, e.g. if the client wants to enable encryption then they also need to set key_uri